data/mods/gen9ssb/abilities.ts

```diff
@@ -727,6 +727,42 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// PartMan
+	ctiershitposter: {
+		shortDesc: "-1 Atk/SpA, +1 Def/SpD. +1 Atk/SpA/Spe, -1 Def/SpD, Mold Breaker if 420+ dmg taken.",
+		name: "C- Tier Shitposter",
+		onDamage(damage, target, source, effect) {
+			target.m.damageTaken ??= 0;
+			target.m.damageTaken += damage;
+			if (target.set && !target.set.shiny) {
+				if (target.m.damageTaken >= 420) {
+					target.set.shiny = true;
+					if (!target.hp) {
+						return this.add(`c:|${getName('PartMan')}|MWAHAHA NOW YOU - oh I'm dead`);
+					}
+					this.add(`c:|${getName('PartMan')}|That's it. Get ready to be rapid-fire hugged.`);
+					target.clearBoosts();
+					this.add('-clearboost', target);
+					this.boost({atk: 1, def: -1, spa: 1, spd: -1, spe: 1});
```

Comment:
Clearboost seems pretty counterintuitive, why not just double the boosts here?

Reply:
I honestly wasn't sure which approach to go with since in case of debuffs/buffs it'd be pretty different, and it doesn't self-boost outside of its ability

Reply:
Oh apparently I suck at asking questions; can I continue with this approach or should I just apply the delta?

Reply:
I leave it up to you, that was just my thought

Reply:
Hmm would it be possible to only reset specific boosts?

Reply:
sorry i forgor. yes you can, use the `-setboost` message, possibly needing a `[silent]` tag because only belly drum/anger point have messages for it